en/docs/develop/creating-artifacts/data-services/creating-data-services.md

```diff
@@ -167,125 +166,129 @@ You can configure the main query details for both SOAP operations and REST resou
 
 	Shown below is an example query with input mapping:
 
-	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/input_mappings.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/input_mappings.png" width="70%"></a>
+	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/input_mappings.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/input_mappings.png" width="80%"></a>
 
-#### Result (Output Mappings)
+#### Result (output mappings)
 
 You can configure output result parameters for the query using this section.
 
 1.  Click **Output Mapping** from the graphical view to edit the output mapping configurations.
 
-	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/out_mapping_expanded.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/out_mapping_expanded.png"></a>
+	<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/out_mapping_expanded.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/out_mapping_expanded.png" width="80%"></a>
 
 2. There are two ways to create the output mapping:
 
-    -	You can use the default configurations to automatically generate the output mappings from the SQL query.
-    -	Alternatively, you can manually add the mappings:
-
-	        1. Click **+ Add Parameter**.
-
-                <a href="{{base_path}}/assets/img/integrate/tutorials/data_services/add_output_mappings.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/add_output_mappings.png" width="50%"></a>
-
-	        2. Enter the following output element details.
-
-			   <table>
-			   <tr>
-			        <th>Property</th>
-			        <th>Description</th>
-			    </tr>
-			<tbody>
-			<tr class="odd">
-			<td>Datasource Type</td>
-			<td>column</td>
-			</tr>
-			<tr class="even">
-			<td>Output Field Name</td>
-			<td>EmployeeNumber</td>
-			</tr>
-			<tr class="odd">
-			<td>Datasource Column Name</td>
-			<td>EmployeeNumber</td>
-			</tr>
-			<tr class="even">
-			<td>Schema Type</td>
-			<td>String</td>
-			</tr>
-			</tbody>
-			</table>
-		 
-		 Follow the same steps to create the remaining output elements.
-
-    Shown below is an example query with output mappings:
-
-    ![]({{base_path}}/assets/img/integrate/tutorials/data_services/output_mapings.png)
+	- You can use the default configurations to automatically generate the output mappings from the SQL query. 
+	- Alternatively, you can manually add the mappings:
+
+      	1. Click **+ Add Parameter**.
+
+	
+			<a href="{{base_path}}/assets/img/integrate/tutorials/data_services/add_output_mappings.png"><img src="{{base_path}}/assets/img/integrate/tutorials/data_services/add_output_mappings.png" width="80%"></a>
+
+      	2. Enter the following output element details.
+
+
+             <table>
+                 <tr>
+                     <th>Property</th>
+                     <th>Description</th>
```

Comment:
Shouldn't this be `Value` instead of `Description`?

Reply:
Yes let's make it as Value